examples/encfs/generatefs/hexstring.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import binascii
+import sys
+
+if len(sys.argv) != 2:
```

Comment:
Just bringing the file back doesn't put the call to run it in the generatefs.sh script